04/index.html

```diff
@@ -1,30 +1,114 @@
 <!DOCTYPE html>
 <html lang="pl">
+
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1.0"
+    >
+    <meta
+        http-equiv="X-UA-Compatible"
+        content="ie=edge"
+    >
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+
+    <script
+        src="https://kit.fontawesome.com/e6dada8901.js"
+        crossorigin="anonymous"
+    ></script>
+
+    <link
+        href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300&display=swap"
+        rel="stylesheet"
+    >
+    <link
+        rel="stylesheet"
+        href="./style/reset.css"
+    />
+
+    <link
+        rel="stylesheet"
+        href="./style/global.css"
+    />
+
+    <link
+        rel="stylesheet"
+        media="(max-width:600px)"
+        href="./style/mobile.css"
+    >
+    <link
+        rel="stylesheet"
+        media="(min-width:601px)"
+        href="./style/desktop.css"
+    >
+
 </head>
+
 <body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
-            </li>
-            <li>
-                <a href="#">oferta</a>
+    <div class="container">
+        <header>
+            <h1 class="logo">BIOLABS</h1>
+            <nav>
+                <label for="menu-switcher"></label>
+                <input
+                    type="checkbox"
+                    id="menu-switcher"
+                >
                 <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
+                    <li class="menu__item">
+                        <a href="#">start</a>
+                    </li>
+                    <li class="menu__item menu__dropdown menu__dropdown--arrow">
+                        <a href="#">oferta</a>
+                        <ul class="dropdown">
+                            <li class="dropdown__item"><a
+                                    href=""
+                                    class="dropdown__link"
+                                >strony internetowe</a></li>
+                            <li class="dropdown__item dropdown__link_2  menu__dropdown--arrow2"><a
+                                    href=""
+                                    class="dropdown__link menu__dropdown"
+                                >pozycjonowanie stron www</a>
+                                <ul class="dropdown__2">
```

Comment:
Ogólnie na wersji mobilej można zrobić na 2 sposoby:
1) pokazać całe menu i zagnieżdżenia oddalić np. od lewej krawędzi (tak było w zadaniu)
2) zrobić `hover` w CSS, gdzie na wersji mobilnej interpretowane jest to jako `click` w ekran - tylko wtedy musisz zrezygnować z `position: absolute` na np. `static` - lepiej to wygląda :)